### ISSUE TITLE
Bug: unexpected ChangeType on original /a_file: !!!

### ISSUE THREAD

**Issue by mbideau** [mbideau/btrfs-diff-go#3]
```
[root@home-server /home/mek101/Progetti/go/bin]# time ./btrfs-diff-go /mountpoints/raid1/anime/data_snapshot.20210816/ /mountpoints/raid1/anime/bk_data_snapshot.2021094/
unexpected ChangeType on original /MyNextLifeAsAVillainess/wget-log: !!!unexpected ChangeType on original /MyNextLifeAsAVillainess/Hamefura_Ep_01_SUB_ITA.mp4: !!!       !!!: /MyNextLifeAsAVillainess/Hamefura_Ep_01_SUB_ITA.mp4
       !!!: /MyNextLifeAsAVillainess/wget-log
     added: /86
     added: /Kobayashi-san Chi no Maid Dragon
     added: /Kyousougiga
     added: /Otome Game no Hametsu Flag shika Nai Akuyaku Reijou ni Tensei shiteshimatta (Hamefura)
...
```
```
495G	data_snapshot.20210816/
535G	bk_data_snapshot.20210904
```
with a time of only!
```
real	0m2,062s
user	0m0,364s
sys	0m2,162s
```
_Originally posted by @Mek101 in #2_

**Comment by mbideau** [comment]
Hey @Mek101,
I have finally fixed all the bugs (see efe8fe3 for this one) and now I am confident in the implementation.
You can check it out again if you want ... that would be great :smiley:

**Comment by Mek101** [comment]
Bug solved👍
